deployments/cloudfoundry/buildpack/bin/supply

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+BUILD_DIR=$1
+CACHE_DIR=$2
+DEPS_DIR=$3
+IDX=$4
+
+BUILDPACK_DIR="$(readlink -f ${BASH_SOURCE%/*})"
+TARGET_DIR="$BUILD_DIR/.otelcollector"
+
+OTEL_CONFIG="${OTEL_CONFIG:-otelconfig.yaml}"
+# Version must be >=0.47.0, the first release included the enabled Cloud Foundry receiver
+OTEL_VERSION="${OTEL_VERSION:-0.47.0}"
```

Comment:
Not sure it needs to be addressed here, but a self updating mechanism is preferable imo (maybe made part of https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/update-deps or something else?) cc @jcheng-splunk

Reply:
Could `OTEL_VERSION` default to `latest` and just use the github api to automatically determine the latest version to download? Something like https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/fpm/common.sh#L77?

Reply:
I asked about this in our team's slack, and @emaderer and I had thought it may be better to stick with a specific version in case of breaking changes. Another concern I had at the time was that I didn't know how to download latest. Now that I have a way to do it that's not as much of a concern.

Do either of you have concerns about possible breaking changes? I'm not sure how much effort it would take to update a hardcoded version, but it would be more tedious than always setting it to latest.

Reply:
Okay, as long as this is a common and supported functionality it should be fine. There's a buildpack variable `OTEL_VERSION`, if that's set we can use the version specified, otherwise use the latest.